### PR TITLE
DOCS-4883 Updates Error Tracking Screenshots 

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Updates screenshots for the Error Tracking page in RUM, adds image width for images in Error Tracking for Backend Services, and standardizes copy across doc pages in RUM, APM, and Logs.

### Motivation

DOCS-4883

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/real_user_monitoring/error_tracking/_index.md [hunk 1]
```diff
@@ -27,9 +27,9 @@ further_reading:
 
 It is critical for your system's health to consistently monitor the errors collected by Datadog. When there are many individual error events, it becomes hard to prioritize errors for troubleshooting. By tracking, triaging, and debugging crashes, you can minimize the impact of fatal errors on your web and mobile applications' user experience.
 
-Once you have set up RUM for **Web and Mobile Apps** error tracking, the issue list populates with cards. You can view open, ignored, or all issues, sort issues by volume or age, and filter issues by all custom and default facets on your web and mobile applications.
+{{< img src="real_user_monitoring/error_tracking/rum_error_tracking.png" alt="The Error Tracking Explorer for RUM displaying issues from your web and mobile application's crash reports" style="width:100%;" >}}
```
Comment (cswatt) [inline content/en/real_user_monitoring/error_tracking/_index.md:30]:
```suggestion
{{< img src="real_user_monitoring/error_tracking/rum_error_tracking.png" alt="The Error Tracking Explorer for RUM displaying issues from your web and mobile applications' crash reports" style="width:100%;" >}}
```

File: content/en/real_user_monitoring/error_tracking/explorer.md [hunk 2]
```diff
@@ -1,17 +1,25 @@
...
+- link: '/real_user_monitoring/error_tracking'
+  tag: 'Documentation'
+  text: 'Learn about Error Tracking for RUM'
 ---
 
-{{< img src="real_user_monitoring/error_tracking/explorer.png" alt="Error Tracking Explorer"  >}}
+## Overview
 
-## Explore your issues
+{{< img src="real_user_monitoring/error_tracking/page.png" alt="The Error Tracking Explorer for RUM displaying issues from your web and mobile application's crash reports" style="width:100%;" >}}
```
Comment (cswatt) [inline content/en/real_user_monitoring/error_tracking/explorer.md:16]:
```suggestion
{{< img src="real_user_monitoring/error_tracking/page.png" alt="The Error Tracking Explorer for RUM displaying issues from your web and mobile applications' crash reports" style="width:100%;" >}}
```

File: content/en/tracing/error_tracking/explorer.md [hunk 3]
```diff
@@ -1,13 +1,25 @@
...
+  text: 'Learn about Error Tracking Monitors'
+- link: '/tracing/error_tracking'
+  tag: 'Documentation'
+  text: 'Learn about Error Tracking for Backend Services'
 ---
 
-{{< img src="tracing/error_tracking/error_tracking_explore_inspect.png" alt="Error Tracking Explorer"  >}}
+## Overview
+
+The Error Tracking Explorer shows you a list of ongoing issues. An _issue_ is a group of similar errors that have the same _fingerprint_ and which might need to be fixed depending on how critical it is.
```
Comment (cswatt) [inline content/en/tracing/error_tracking/explorer.md:16]:
```suggestion
The Error Tracking Explorer shows you a list of ongoing issues. An _issue_ is a group of similar errors that have the same _fingerprint_ and which might need to be fixed depending on how critical they are. 
```

File: content/en/real_user_monitoring/error_tracking/_index.md [hunk 4]
```diff
@@ -27,9 +27,9 @@ further_reading:
...
 
-{{< img src="real_user_monitoring/error_tracking/page.png" alt="The Error Tracking Explorer for RUM displaying issues from your web and mobile application's crash reports" style="width:100%;" >}}
+Once you have set up RUM for **Web and Mobile Apps** error tracking, the issue list populates with cards. You can view open, ignored, or all issues, sort issues by volume or age, and filter issues by all custom and default facets on your web and mobile applications.
```
Comment (cswatt) [inline content/en/real_user_monitoring/error_tracking/_index.md:32]:
```suggestion
Once you have set up RUM for **Web and Mobile Apps** error tracking, the issue list populates with cards. You can view open, ignored, or all issues; sort issues by volume or age; and filter issues by all custom and default facets on your web and mobile applications.
```

**Review by cswatt — CHANGES_REQUESTED** [review]
just some grammar nits, otherwise looks good!

**Review by cswatt — APPROVED** [review]

File: content/en/tracing/error_tracking/explorer.md [hunk 5]
```diff
@@ -53,5 +65,10 @@ Each error sample gives the information you need to understand why it happened a
 - All the error span tags, such as the resource or operation name, with direct access to the related trace or to logs that are linked to it.
 - Insights about the health of the underlying host or container when this particular error occurred.
 
+## Further Reading
+
+{{< partial name="whats-next/whats-next.html" >}}
+
 [1]: /tracing/trace_explorer/#live-search-for-15-minutes
 [2]: /tracing/trace_pipeline/trace_retention/#retention-filters
+[3]: https://app.datadoghq.com/rum/error-tracking
```
Comment (alai97) [inline content/en/tracing/error_tracking/explorer.md:74]:
```suggestion
[3]: https://app.datadoghq.com/apm/error-tracking
```

File: content/en/tracing/error_tracking/explorer.md [hunk 6]
```diff
@@ -1,13 +1,25 @@
...
+## Overview
+
+The Error Tracking Explorer shows you a list of ongoing issues. An _issue_ is a group of similar errors that have the same _fingerprint_ and which might need to be fixed depending on how critical they are. 
+
+{{< img src="tracing/error_tracking/error_tracking_explore_inspect.png" alt="Error Tracking Explorer" style="width:100%;" >}}
 
 ## Explore your issues
 
-The Error Tracking Explorer shows you a list of ongoing issues. An _issue_ is a group of similar errors that have the same _fingerprint_ and which might need to be fixed depending on how critical it is. Each item listed in the explorer surfaces high-level information about the issue:
+Each item listed in the Error Tracking Explorer surfaces high-level information about the issue, including the following:
```
Comment (alai97) [inline content/en/tracing/error_tracking/explorer.md:22]:
```suggestion
Each item listed in the [Error Tracking Explorer][3] surfaces high-level information about the issue, including the following:
```

File: content/en/real_user_monitoring/error_tracking/_index.md [hunk 7]
```diff
@@ -29,9 +29,9 @@ algolia:
 
 It is critical for your system's health to consistently monitor the errors collected by Datadog. When there are many individual error events, it becomes hard to prioritize errors for troubleshooting. By tracking, triaging, and debugging crashes, you can minimize the impact of fatal errors on your web and mobile applications' user experience.
```
Comment (rtrieu) [inline content/en/real_user_monitoring/error_tracking/_index.md:31]: Should we mention somewhere here how to navigate to the Error Tracking page from UX Monitoring or elsewhere?

**Review by rtrieu — CHANGES_REQUESTED** [review]
This looks good, except the new image doesn't appear (I'm still seeing the old cached one). Any chance this is because the filename wasn't changed to something like `rum_error_tracking_1.png`?

File: content/en/real_user_monitoring/error_tracking/_index.md [hunk 8]
```diff
@@ -29,9 +29,9 @@ algolia:
...
 
-Once you have set up RUM for **Web and Mobile Apps** error tracking, the issue list populates with cards. You can view open, ignored, or all issues, sort issues by volume or age, and filter issues by all custom and default facets on your web and mobile applications.
+{{< img src="real_user_monitoring/error_tracking/rum_error_tracking.png" alt="The Error Tracking Explorer for RUM displaying issues from your web and mobile applications' crash reports" style="width:100%;" >}}
```
Comment (rtrieu) [inline content/en/real_user_monitoring/error_tracking/_index.md:32]:
```suggestion
{{< img src="real_user_monitoring/error_tracking/rum_error_tracking_1.png" alt="The Error Tracking Explorer for RUM displaying issues from your web and mobile applications' crash reports" style="width:100%;" >}}
```

**Review by rtrieu — CHANGES_REQUESTED** [review]
One last thing..

**Review by rtrieu — APPROVED** [review]
Great!

**Review by nhientrandatadog — APPROVED** [review]

**Pull request merged**